csharp/ql/test/library-tests/dataflow/library/FlowSummariesFiltered.expected

```diff
@@ -327,8 +323,6 @@
 | System.Collections.Generic;IList<>;true;get_Item;(System.Int32);;Argument[Qualifier].Element;ReturnValue;value;manual |
 | System.Collections.Generic;IList<>;true;set_Item;(System.Int32,T);;Argument[1];Argument[Qualifier].Element;value;manual |
 | System.Collections.Generic;ISet<>;true;Add;(T);;Argument[0];Argument[Qualifier].Element;value;manual |
-| System.Collections.Generic;KeyValuePair;false;Create<,>;(TKey,TValue);;Argument[0];ReturnValue;taint;generated |
-| System.Collections.Generic;KeyValuePair;false;Create<,>;(TKey,TValue);;Argument[1];ReturnValue;taint;generated |
```

Comment:
These also seem to be correct flows. Why are these removed? Without checking, I expect the `Create<,>` method to be quite simple, so we should be able to identify flow in there.

Reply:
It appears that these rows have disappered since we no longer use the source code for the KeyValuePair constructor as there are CSV rows it.
The rows look like
```
"System.Collections.Generic;KeyValuePair<,>;false;KeyValuePair;(TKey,TValue);;Argument[0];ReturnValue.Property[System.Collections.Generic.KeyValuePair<,>.Key];value;manual",
"System.Collections.Generic;KeyValuePair<,>;false;KeyValuePair;(TKey,TValue);;Argument[1];ReturnValue.Property[System.Collections.Generic.KeyValuePair<,>.Value];value;manual"
```

That is, based on these rows the model generator no longer yields any results for
```csharp
    public static KeyValuePair<TKey, TValue> Create<TKey, TValue>(TKey key, TValue value) =>
        new KeyValuePair<TKey, TValue>(key, value);
```
but a summary will be derived for 
```csharp
    public static TKey Create2<TKey, TValue>(TKey key, TValue value) =>
        new KeyValuePair<TKey, TValue>(key, value).Key;
```